FrontEnd/src/pages/Browse.tsx

```diff
@@ -13,47 +13,72 @@ import {
 import DownloadIcon from '@mui/icons-material/Download';
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
-import data from '../mock-table-data.json';
 import CircularProgress from '@mui/material/CircularProgress';
 import '../CSS/Browse.css';
+import SelectPlot from '../components/SelectPlot';
+import { plotProps } from '../components/SelectPlot';
 
-const Browse = () => {
-  let handleRemove = (i: any) => {
-    const newRows = [...rows];
-    const index = rows.findIndex((row) => row.file === i);
-    newRows.splice(index, 1);
-    setRows(newRows);
+interface FileWithMetadata {
+  fileName: {
+    metaData: string;
   };
+}
 
-  async function getData() {
-    return new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve(data);
-        //reject('Failed to load');
-      }, 2000);
-    });
-  }
+const Browse = (props: plotProps) => {
+  // @TODO - implement remove and download files
```

Comment:
No need to have this TODO here. Better to have it as GitHub issue(s) to make it easier to find/track